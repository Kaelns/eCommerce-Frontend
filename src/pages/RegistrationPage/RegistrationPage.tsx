import { Stack } from '@mui/system';
import { Chip, Button, Divider } from '@mui/material';

import { router } from '@/app';

import { TitleTypography } from '@/shared/ui/elements/typography/TitleTypography';

import { Paths } from '@/shared/model/data/enums';
// import { RegistrationForm } from '@/features/components/AuthForms/RegistrationForm/RegistrationForm';

export function RegistrationPage() {
  const navigateToLogin = () => router.navigate(Paths.LOGIN);

  return (
    <Stack gap={2} alignItems="center">
      <Stack gap={1.5} width={{ zero: 1, tablet: 400 }}>
        <TitleTypography textAlign="center">Registration Page</TitleTypography>
        <Divider>
          <Chip label="Or if you are already registered:" />
        </Divider>
        <Button variant="contained" onClick={navigateToLogin}>
          Login
        </Button>
      </Stack>
      {/* <RegistrationForm /> */}
    </Stack>
  );
}
