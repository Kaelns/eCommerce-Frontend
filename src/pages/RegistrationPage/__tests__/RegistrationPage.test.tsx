import { RegistrationPage } from '@/pages/RegistrationPage/RegistrationPage';
import { render } from '@testing-library/react';

describe('Given RegistrationPage component', () => {
  it('When rendered, should match snapshot', () => {
    const { asFragment } = render(<RegistrationPage />);
    expect(asFragment()).toMatchSnapshot();
  });
});
