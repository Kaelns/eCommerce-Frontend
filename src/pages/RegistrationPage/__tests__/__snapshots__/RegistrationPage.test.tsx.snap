// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Given RegistrationPage component > When rendered, should match snapshot 1`] = `
<DocumentFragment>
  <div
    class="MuiStack-root css-m8u4h0-MuiStack-root"
  >
    <div
      class="MuiStack-root css-z5b4yp-MuiStack-root"
    >
      <h5
        class="MuiTypography-root MuiTypography-h5 css-1mcnxdd-MuiTypography-root"
      >
        Registration Page
      </h5>
      <div
        aria-orientation="horizontal"
        class="MuiDivider-root MuiDivider-fullWidth MuiDivider-withChildren css-1tscyog-MuiDivider-root"
        role="separator"
      >
        <span
          class="MuiDivider-wrapper css-qywfm8-MuiDivider-wrapper"
        >
          <div
            class="MuiChip-root MuiChip-filled MuiChip-sizeMedium MuiChip-colorDefault MuiChip-filledDefault css-1sl1ukt-MuiChip-root"
          >
            <span
              class="MuiChip-label MuiChip-labelMedium css-6od3lo-MuiChip-label"
            >
              Or if you are already registered:
            </span>
          </div>
        </span>
      </div>
      <button
        class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-colorPrimary MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-colorPrimary css-142372x-MuiButtonBase-root-MuiButton-root"
        tabindex="0"
        type="button"
      >
        Login
      </button>
    </div>
    <form
      class="MuiStack-root css-1cnp5lb-MuiStack-root"
    >
      <h6
        class="MuiTypography-root MuiTypography-h6 css-1sov8tp-MuiTypography-root"
      >
        User credentials
      </h6>
      <label
        class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-animated MuiFormLabel-colorPrimary Mui-required MuiInputLabel-root MuiInputLabel-animated css-1d2t4i-MuiFormLabel-root-MuiInputLabel-root"
      >
        First name
        <span
          aria-hidden="true"
          class="MuiFormLabel-asterisk MuiInputLabel-asterisk css-wgai2y-MuiFormLabel-asterisk"
        >
           *
        </span>
      </label>
      <div
        class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-colorPrimary css-j882ge-MuiInputBase-root-MuiOutlinedInput-root"
        required=""
      >
        <input
          aria-invalid="false"
          class="MuiInputBase-input MuiOutlinedInput-input css-1jk99ih-MuiInputBase-input-MuiOutlinedInput-input"
          name="firstName"
          required=""
          type="text"
          value=""
        />
        <fieldset
          aria-hidden="true"
          class="MuiOutlinedInput-notchedOutline css-1d3z3hw-MuiOutlinedInput-notchedOutline"
        >
          <legend
            class="css-gpjuor"
          >
            <span
              class="notranslate"
            >
              ​
            </span>
          </legend>
        </fieldset>
      </div>
      <p
        class="MuiFormHelperText-root Mui-error css-1d1r5q-MuiFormHelperText-root"
      />
      <label
        class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-animated MuiFormLabel-colorPrimary Mui-required MuiInputLabel-root MuiInputLabel-animated css-1d2t4i-MuiFormLabel-root-MuiInputLabel-root"
      >
        Last name
        <span
          aria-hidden="true"
          class="MuiFormLabel-asterisk MuiInputLabel-asterisk css-wgai2y-MuiFormLabel-asterisk"
        >
           *
        </span>
      </label>
      <div
        class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-colorPrimary css-j882ge-MuiInputBase-root-MuiOutlinedInput-root"
        required=""
      >
        <input
          aria-invalid="false"
          class="MuiInputBase-input MuiOutlinedInput-input css-1jk99ih-MuiInputBase-input-MuiOutlinedInput-input"
          name="lastName"
          required=""
          type="text"
          value=""
        />
        <fieldset
          aria-hidden="true"
          class="MuiOutlinedInput-notchedOutline css-1d3z3hw-MuiOutlinedInput-notchedOutline"
        >
          <legend
            class="css-gpjuor"
          >
            <span
              class="notranslate"
            >
              ​
            </span>
          </legend>
        </fieldset>
      </div>
      <p
        class="MuiFormHelperText-root Mui-error css-1d1r5q-MuiFormHelperText-root"
      />
      <label
        class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-animated MuiFormLabel-colorPrimary MuiInputLabel-root MuiInputLabel-animated css-1d2t4i-MuiFormLabel-root-MuiInputLabel-root"
      >
        Birthday: 
      </label>
      <div
        class="MuiFormControl-root MuiTextField-root css-z3c6am-MuiFormControl-root-MuiTextField-root"
      >
        <div
          class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-colorPrimary MuiInputBase-formControl MuiInputBase-adornedEnd Mui-readOnly MuiInputBase-readOnly css-lxfshk-MuiInputBase-root-MuiOutlinedInput-root"
        >
          <input
            aria-invalid="false"
            aria-label="Choose date, selected date is Sep 26, 2011"
            autocomplete="off"
            class="MuiInputBase-input MuiOutlinedInput-input MuiInputBase-inputAdornedEnd Mui-readOnly MuiInputBase-readOnly css-lc42l8-MuiInputBase-input-MuiOutlinedInput-input"
            id=":r1:"
            inputmode="text"
            placeholder="DD/MM/YYYY"
            readonly=""
            type="text"
            value="26/09/2011"
          />
          <fieldset
            aria-hidden="true"
            class="MuiOutlinedInput-notchedOutline css-1d3z3hw-MuiOutlinedInput-notchedOutline"
          >
            <legend
              class="css-gpjuor"
            >
              <span
                class="notranslate"
              >
                ​
              </span>
            </legend>
          </fieldset>
        </div>
      </div>
      <p
        class="MuiFormHelperText-root Mui-error css-1d1r5q-MuiFormHelperText-root"
      />
      <label
        class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-animated MuiFormLabel-colorPrimary Mui-required MuiInputLabel-root MuiInputLabel-animated css-1d2t4i-MuiFormLabel-root-MuiInputLabel-root"
      >
        Enter your email
        <span
          aria-hidden="true"
          class="MuiFormLabel-asterisk MuiInputLabel-asterisk css-wgai2y-MuiFormLabel-asterisk"
        >
           *
        </span>
      </label>
      <div
        class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-colorPrimary css-j882ge-MuiInputBase-root-MuiOutlinedInput-root"
        required=""
      >
        <input
          aria-invalid="false"
          class="MuiInputBase-input MuiOutlinedInput-input css-1jk99ih-MuiInputBase-input-MuiOutlinedInput-input"
          name="email"
          required=""
          type="text"
          value=""
        />
        <fieldset
          aria-hidden="true"
          class="MuiOutlinedInput-notchedOutline css-1d3z3hw-MuiOutlinedInput-notchedOutline"
        >
          <legend
            class="css-gpjuor"
          >
            <span
              class="notranslate"
            >
              ​
            </span>
          </legend>
        </fieldset>
      </div>
      <p
        class="MuiFormHelperText-root Mui-error css-1d1r5q-MuiFormHelperText-root"
      />
      <label
        class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-animated MuiFormLabel-colorPrimary Mui-required MuiInputLabel-root MuiInputLabel-animated css-1d2t4i-MuiFormLabel-root-MuiInputLabel-root"
      >
        Enter your password
        <span
          aria-hidden="true"
          class="MuiFormLabel-asterisk MuiInputLabel-asterisk css-wgai2y-MuiFormLabel-asterisk"
        >
           *
        </span>
      </label>
      <div
        class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-colorPrimary MuiInputBase-adornedEnd css-lxfshk-MuiInputBase-root-MuiOutlinedInput-root"
        required=""
      >
        <input
          aria-invalid="false"
          class="MuiInputBase-input MuiOutlinedInput-input MuiInputBase-inputAdornedEnd css-lc42l8-MuiInputBase-input-MuiOutlinedInput-input"
          name="password"
          required=""
          type="password"
          value=""
        />
        <button
          class="MuiButtonBase-root MuiIconButton-root MuiIconButton-edgeEnd MuiIconButton-sizeMedium MuiInputAdornment-root MuiInputAdornment-positionEnd css-1yrfzg2-MuiButtonBase-root-MuiIconButton-root-MuiInputAdornment-root"
          tabindex="0"
          type="button"
        >
          <svg
            aria-hidden="true"
            class="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall css-18w7uxr-MuiSvgIcon-root"
            data-testid="VisibilityIcon"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M12 4.5C7 4.5 2.73 7.61 1 12c1.73 4.39 6 7.5 11 7.5s9.27-3.11 11-7.5c-1.73-4.39-6-7.5-11-7.5M12 17c-2.76 0-5-2.24-5-5s2.24-5 5-5 5 2.24 5 5-2.24 5-5 5m0-8c-1.66 0-3 1.34-3 3s1.34 3 3 3 3-1.34 3-3-1.34-3-3-3"
            />
          </svg>
        </button>
        <fieldset
          aria-hidden="true"
          class="MuiOutlinedInput-notchedOutline css-1d3z3hw-MuiOutlinedInput-notchedOutline"
        >
          <legend
            class="css-gpjuor"
          >
            <span
              class="notranslate"
            >
              ​
            </span>
          </legend>
        </fieldset>
      </div>
      <p
        class="MuiFormHelperText-root Mui-error css-1d1r5q-MuiFormHelperText-root"
      />
      <h6
        class="MuiTypography-root MuiTypography-h6 css-1g7aky8-MuiTypography-root"
      >
        Shipping Address
      </h6>
      <label
        class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-animated MuiFormLabel-colorPrimary MuiInputLabel-root MuiInputLabel-animated css-1d2t4i-MuiFormLabel-root-MuiInputLabel-root"
      >
        Country: 
      </label>
      <div
        class="MuiAutocomplete-root MuiAutocomplete-hasClearIcon MuiAutocomplete-hasPopupIcon css-ka7ti6-MuiAutocomplete-root"
      >
        <div
          class="MuiFormControl-root MuiFormControl-fullWidth MuiTextField-root css-y4n9bg-MuiFormControl-root-MuiTextField-root"
        >
          <div
            class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-colorPrimary MuiInputBase-fullWidth MuiInputBase-formControl MuiInputBase-adornedEnd MuiAutocomplete-inputRoot css-1wgn40h-MuiInputBase-root-MuiOutlinedInput-root"
          >
            <input
              aria-autocomplete="list"
              aria-expanded="false"
              aria-invalid="false"
              autocapitalize="none"
              autocomplete="off"
              class="MuiInputBase-input MuiOutlinedInput-input MuiInputBase-inputAdornedEnd MuiAutocomplete-input MuiAutocomplete-inputFocused css-lc42l8-MuiInputBase-input-MuiOutlinedInput-input"
              id="shipping"
              name="shippingCountry"
              role="combobox"
              spellcheck="false"
              type="text"
              value="United States"
            />
            <div
              class="MuiAutocomplete-endAdornment css-p1olib-MuiAutocomplete-endAdornment"
            >
              <button
                aria-label="Clear"
                class="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeMedium MuiAutocomplete-clearIndicator css-1pkh4lu-MuiButtonBase-root-MuiIconButton-root-MuiAutocomplete-clearIndicator"
                tabindex="-1"
                title="Clear"
                type="button"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall css-18w7uxr-MuiSvgIcon-root"
                  data-testid="CloseIcon"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M19 6.41L17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"
                  />
                </svg>
              </button>
              <button
                aria-label="Open"
                class="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeMedium MuiAutocomplete-popupIndicator css-2anpn7-MuiButtonBase-root-MuiIconButton-root-MuiAutocomplete-popupIndicator"
                tabindex="-1"
                title="Open"
                type="button"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-20bmp1-MuiSvgIcon-root"
                  data-testid="ArrowDropDownIcon"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M7 10l5 5 5-5z"
                  />
                </svg>
              </button>
            </div>
            <fieldset
              aria-hidden="true"
              class="MuiOutlinedInput-notchedOutline css-1d3z3hw-MuiOutlinedInput-notchedOutline"
            >
              <legend
                class="css-gpjuor"
              >
                <span
                  class="notranslate"
                >
                  ​
                </span>
              </legend>
            </fieldset>
          </div>
        </div>
      </div>
      <label
        class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-animated MuiFormLabel-colorPrimary Mui-required MuiInputLabel-root MuiInputLabel-animated css-1d2t4i-MuiFormLabel-root-MuiInputLabel-root"
      >
        Postal code
        <span
          aria-hidden="true"
          class="MuiFormLabel-asterisk MuiInputLabel-asterisk css-wgai2y-MuiFormLabel-asterisk"
        >
           *
        </span>
      </label>
      <div
        class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-colorPrimary css-j882ge-MuiInputBase-root-MuiOutlinedInput-root"
        required=""
      >
        <input
          class="MuiInputBase-input MuiOutlinedInput-input css-1jk99ih-MuiInputBase-input-MuiOutlinedInput-input"
          name="shippingPostalCode"
          required=""
          type="text"
          value=""
        />
        <fieldset
          aria-hidden="true"
          class="MuiOutlinedInput-notchedOutline css-1d3z3hw-MuiOutlinedInput-notchedOutline"
        >
          <legend
            class="css-gpjuor"
          >
            <span
              class="notranslate"
            >
              ​
            </span>
          </legend>
        </fieldset>
      </div>
      <p
        class="MuiFormHelperText-root Mui-error css-1d1r5q-MuiFormHelperText-root"
      />
      <label
        class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-animated MuiFormLabel-colorPrimary Mui-required MuiInputLabel-root MuiInputLabel-animated css-1d2t4i-MuiFormLabel-root-MuiInputLabel-root"
      >
        City
        <span
          aria-hidden="true"
          class="MuiFormLabel-asterisk MuiInputLabel-asterisk css-wgai2y-MuiFormLabel-asterisk"
        >
           *
        </span>
      </label>
      <div
        class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-colorPrimary css-j882ge-MuiInputBase-root-MuiOutlinedInput-root"
        required=""
      >
        <input
          class="MuiInputBase-input MuiOutlinedInput-input css-1jk99ih-MuiInputBase-input-MuiOutlinedInput-input"
          name="shippingCity"
          required=""
          type="text"
          value=""
        />
        <fieldset
          aria-hidden="true"
          class="MuiOutlinedInput-notchedOutline css-1d3z3hw-MuiOutlinedInput-notchedOutline"
        >
          <legend
            class="css-gpjuor"
          >
            <span
              class="notranslate"
            >
              ​
            </span>
          </legend>
        </fieldset>
      </div>
      <p
        class="MuiFormHelperText-root Mui-error css-1d1r5q-MuiFormHelperText-root"
      />
      <label
        class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-animated MuiFormLabel-colorPrimary Mui-required MuiInputLabel-root MuiInputLabel-animated css-1d2t4i-MuiFormLabel-root-MuiInputLabel-root"
      >
        Street
        <span
          aria-hidden="true"
          class="MuiFormLabel-asterisk MuiInputLabel-asterisk css-wgai2y-MuiFormLabel-asterisk"
        >
           *
        </span>
      </label>
      <div
        class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-colorPrimary css-j882ge-MuiInputBase-root-MuiOutlinedInput-root"
        required=""
      >
        <input
          class="MuiInputBase-input MuiOutlinedInput-input css-1jk99ih-MuiInputBase-input-MuiOutlinedInput-input"
          name="shippingStreet"
          required=""
          type="text"
          value=""
        />
        <fieldset
          aria-hidden="true"
          class="MuiOutlinedInput-notchedOutline css-1d3z3hw-MuiOutlinedInput-notchedOutline"
        >
          <legend
            class="css-gpjuor"
          >
            <span
              class="notranslate"
            >
              ​
            </span>
          </legend>
        </fieldset>
      </div>
      <p
        class="MuiFormHelperText-root Mui-error css-1d1r5q-MuiFormHelperText-root"
      />
      <label
        class="MuiFormControlLabel-root MuiFormControlLabel-labelPlacementEnd css-j204z7-MuiFormControlLabel-root"
      >
        <span
          class="MuiButtonBase-root MuiCheckbox-root MuiCheckbox-colorPrimary MuiCheckbox-sizeMedium PrivateSwitchBase-root MuiCheckbox-root MuiCheckbox-colorPrimary MuiCheckbox-sizeMedium MuiCheckbox-root MuiCheckbox-colorPrimary MuiCheckbox-sizeMedium css-1y5vrnu-MuiButtonBase-root-MuiCheckbox-root"
        >
          <input
            class="PrivateSwitchBase-input css-1m9pwf3"
            data-indeterminate="false"
            type="checkbox"
          />
          <svg
            aria-hidden="true"
            class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-20bmp1-MuiSvgIcon-root"
            data-testid="CheckBoxOutlineBlankIcon"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"
            />
          </svg>
        </span>
        <span
          class="MuiTypography-root MuiTypography-body1 MuiFormControlLabel-label css-1edfpdg-MuiTypography-root"
        >
          Set as billing address
        </span>
      </label>
      <label
        class="MuiFormControlLabel-root MuiFormControlLabel-labelPlacementEnd css-j204z7-MuiFormControlLabel-root"
      >
        <span
          class="MuiButtonBase-root MuiCheckbox-root MuiCheckbox-colorPrimary MuiCheckbox-sizeMedium PrivateSwitchBase-root MuiCheckbox-root MuiCheckbox-colorPrimary MuiCheckbox-sizeMedium MuiCheckbox-root MuiCheckbox-colorPrimary MuiCheckbox-sizeMedium css-1y5vrnu-MuiButtonBase-root-MuiCheckbox-root"
        >
          <input
            class="PrivateSwitchBase-input css-1m9pwf3"
            data-indeterminate="false"
            type="checkbox"
          />
          <svg
            aria-hidden="true"
            class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-20bmp1-MuiSvgIcon-root"
            data-testid="CheckBoxOutlineBlankIcon"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"
            />
          </svg>
        </span>
        <span
          class="MuiTypography-root MuiTypography-body1 MuiFormControlLabel-label css-1edfpdg-MuiTypography-root"
        >
          Set as default shipping address
        </span>
      </label>
      <h6
        class="MuiTypography-root MuiTypography-h6 css-1g7aky8-MuiTypography-root"
      >
        Billing Address
      </h6>
      <label
        class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-animated MuiFormLabel-colorPrimary MuiInputLabel-root MuiInputLabel-animated css-1d2t4i-MuiFormLabel-root-MuiInputLabel-root"
      >
        Country: 
      </label>
      <div
        class="MuiAutocomplete-root MuiAutocomplete-hasClearIcon MuiAutocomplete-hasPopupIcon css-ka7ti6-MuiAutocomplete-root"
      >
        <div
          class="MuiFormControl-root MuiFormControl-fullWidth MuiTextField-root css-y4n9bg-MuiFormControl-root-MuiTextField-root"
        >
          <div
            class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-colorPrimary MuiInputBase-fullWidth MuiInputBase-formControl MuiInputBase-adornedEnd MuiAutocomplete-inputRoot css-1wgn40h-MuiInputBase-root-MuiOutlinedInput-root"
          >
            <input
              aria-autocomplete="list"
              aria-expanded="false"
              aria-invalid="false"
              autocapitalize="none"
              autocomplete="off"
              class="MuiInputBase-input MuiOutlinedInput-input MuiInputBase-inputAdornedEnd MuiAutocomplete-input MuiAutocomplete-inputFocused css-lc42l8-MuiInputBase-input-MuiOutlinedInput-input"
              id="billing"
              name="billingCountry"
              role="combobox"
              spellcheck="false"
              type="text"
              value="United States"
            />
            <div
              class="MuiAutocomplete-endAdornment css-p1olib-MuiAutocomplete-endAdornment"
            >
              <button
                aria-label="Clear"
                class="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeMedium MuiAutocomplete-clearIndicator css-1pkh4lu-MuiButtonBase-root-MuiIconButton-root-MuiAutocomplete-clearIndicator"
                tabindex="-1"
                title="Clear"
                type="button"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall css-18w7uxr-MuiSvgIcon-root"
                  data-testid="CloseIcon"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M19 6.41L17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"
                  />
                </svg>
              </button>
              <button
                aria-label="Open"
                class="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeMedium MuiAutocomplete-popupIndicator css-2anpn7-MuiButtonBase-root-MuiIconButton-root-MuiAutocomplete-popupIndicator"
                tabindex="-1"
                title="Open"
                type="button"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-20bmp1-MuiSvgIcon-root"
                  data-testid="ArrowDropDownIcon"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M7 10l5 5 5-5z"
                  />
                </svg>
              </button>
            </div>
            <fieldset
              aria-hidden="true"
              class="MuiOutlinedInput-notchedOutline css-1d3z3hw-MuiOutlinedInput-notchedOutline"
            >
              <legend
                class="css-gpjuor"
              >
                <span
                  class="notranslate"
                >
                  ​
                </span>
              </legend>
            </fieldset>
          </div>
        </div>
      </div>
      <label
        class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-animated MuiFormLabel-colorPrimary Mui-required MuiInputLabel-root MuiInputLabel-animated css-1d2t4i-MuiFormLabel-root-MuiInputLabel-root"
      >
        Postal code
        <span
          aria-hidden="true"
          class="MuiFormLabel-asterisk MuiInputLabel-asterisk css-wgai2y-MuiFormLabel-asterisk"
        >
           *
        </span>
      </label>
      <div
        class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-colorPrimary css-j882ge-MuiInputBase-root-MuiOutlinedInput-root"
        required=""
      >
        <input
          class="MuiInputBase-input MuiOutlinedInput-input css-1jk99ih-MuiInputBase-input-MuiOutlinedInput-input"
          name="billingPostalCode"
          required=""
          type="text"
          value=""
        />
        <fieldset
          aria-hidden="true"
          class="MuiOutlinedInput-notchedOutline css-1d3z3hw-MuiOutlinedInput-notchedOutline"
        >
          <legend
            class="css-gpjuor"
          >
            <span
              class="notranslate"
            >
              ​
            </span>
          </legend>
        </fieldset>
      </div>
      <p
        class="MuiFormHelperText-root Mui-error css-1d1r5q-MuiFormHelperText-root"
      />
      <label
        class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-animated MuiFormLabel-colorPrimary Mui-required MuiInputLabel-root MuiInputLabel-animated css-1d2t4i-MuiFormLabel-root-MuiInputLabel-root"
      >
        City
        <span
          aria-hidden="true"
          class="MuiFormLabel-asterisk MuiInputLabel-asterisk css-wgai2y-MuiFormLabel-asterisk"
        >
           *
        </span>
      </label>
      <div
        class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-colorPrimary css-j882ge-MuiInputBase-root-MuiOutlinedInput-root"
        required=""
      >
        <input
          class="MuiInputBase-input MuiOutlinedInput-input css-1jk99ih-MuiInputBase-input-MuiOutlinedInput-input"
          name="billingCity"
          required=""
          type="text"
          value=""
        />
        <fieldset
          aria-hidden="true"
          class="MuiOutlinedInput-notchedOutline css-1d3z3hw-MuiOutlinedInput-notchedOutline"
        >
          <legend
            class="css-gpjuor"
          >
            <span
              class="notranslate"
            >
              ​
            </span>
          </legend>
        </fieldset>
      </div>
      <p
        class="MuiFormHelperText-root Mui-error css-1d1r5q-MuiFormHelperText-root"
      />
      <label
        class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-animated MuiFormLabel-colorPrimary Mui-required MuiInputLabel-root MuiInputLabel-animated css-1d2t4i-MuiFormLabel-root-MuiInputLabel-root"
      >
        Street
        <span
          aria-hidden="true"
          class="MuiFormLabel-asterisk MuiInputLabel-asterisk css-wgai2y-MuiFormLabel-asterisk"
        >
           *
        </span>
      </label>
      <div
        class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-colorPrimary css-j882ge-MuiInputBase-root-MuiOutlinedInput-root"
        required=""
      >
        <input
          class="MuiInputBase-input MuiOutlinedInput-input css-1jk99ih-MuiInputBase-input-MuiOutlinedInput-input"
          name="billingStreet"
          required=""
          type="text"
          value=""
        />
        <fieldset
          aria-hidden="true"
          class="MuiOutlinedInput-notchedOutline css-1d3z3hw-MuiOutlinedInput-notchedOutline"
        >
          <legend
            class="css-gpjuor"
          >
            <span
              class="notranslate"
            >
              ​
            </span>
          </legend>
        </fieldset>
      </div>
      <p
        class="MuiFormHelperText-root Mui-error css-1d1r5q-MuiFormHelperText-root"
      />
      <label
        class="MuiFormControlLabel-root MuiFormControlLabel-labelPlacementEnd css-j204z7-MuiFormControlLabel-root"
      >
        <span
          class="MuiButtonBase-root MuiCheckbox-root MuiCheckbox-colorPrimary MuiCheckbox-sizeMedium PrivateSwitchBase-root MuiCheckbox-root MuiCheckbox-colorPrimary MuiCheckbox-sizeMedium MuiCheckbox-root MuiCheckbox-colorPrimary MuiCheckbox-sizeMedium css-1y5vrnu-MuiButtonBase-root-MuiCheckbox-root"
        >
          <input
            class="PrivateSwitchBase-input css-1m9pwf3"
            data-indeterminate="false"
            type="checkbox"
          />
          <svg
            aria-hidden="true"
            class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-20bmp1-MuiSvgIcon-root"
            data-testid="CheckBoxOutlineBlankIcon"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"
            />
          </svg>
        </span>
        <span
          class="MuiTypography-root MuiTypography-body1 MuiFormControlLabel-label css-1edfpdg-MuiTypography-root"
        >
          Set as default billing address
        </span>
      </label>
      <button
        class="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-colorPrimary Mui-disabled MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-colorPrimary css-29r4wd-MuiButtonBase-root-MuiButton-root"
        disabled=""
        tabindex="-1"
        type="submit"
      >
        Register
      </button>
    </form>
  </div>
</DocumentFragment>
`;
